core/templates/pages/story-viewer-page/story-viewer-page.mainpage.html

```diff
@@ -77,6 +77,37 @@
         width: 100px;
       }
 
+      .lesson-icon-completed {
```

Comment:
Always start with `oppia-` for class names (see guidelines [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).

Reply:
Done.